### PR TITLE
Support for French Pokémon names

### PR DESCRIPTION
**Pull request by bennyn**

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -1 +1 @@
-* text=auto
+* text eol=lf
```
Comment (sindresorhus) [inline .gitattributes]: Can't do that. It makes even binary be text.

The correct way is to set explicit file extensions to be `lf`.

File: readme.md [hunk 2]
```diff
@@ -53,6 +53,8 @@ pokemon.getName(100);
 //=> 'Voltorb'
 pokemon.getName(100, 'de');
 //=> 'Voltobal'
+pokemon.getName(100, 'fr');
```
Comment (sindresorhus) [inline readme.md]: We don't need more language examples. It's already clear that you can pass a language code.

File: test.js [hunk 3]
```diff
@@ -47,6 +53,12 @@ test('Get ID by English name', testIDs, 'en', [
 	'Volcanion'
 ]);
 
+test('Get ID by French name', testIDs, 'fr', [
```
Comment (sindresorhus) [inline test.js]: We don't need this test for every new language. It's enough to test the `Get French name by ID`. Everything else is already tested by English and German.

File: .gitattributes (hunk 1 above)
Comment (sindresorhus) [inline .gitattributes]: See https://github.com/sindresorhus/generator-nm/blob/1f5ff89c082b8e070d025597a87adc96462f406e/app/templates/gitattributes#L2

**Comment by bennyn** [comment]
Ok, I got the point. :smiley: Thanks for reviewing, @sindresorhus.

**Comment by sindresorhus** [comment]
Looks good :)

**Pull request merged**